src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala

```diff
@@ -1,18 +1,21 @@
 package org.broadinstitute.dsde.workbench.leonardo.service
 
 import akka.actor.ActorSystem
+import akka.testkit.TestKit
 import com.typesafe.config.ConfigFactory
 import net.ceedubs.ficus.Ficus._
 import org.broadinstitute.dsde.workbench.leonardo.config.DataprocConfig
 import org.broadinstitute.dsde.workbench.leonardo.dao.GoogleDataprocDAO
 import org.broadinstitute.dsde.workbench.leonardo.db.DbSingleton
-import org.scalatest.{FlatSpec, Matchers}
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, FlatSpecLike, Matchers}
 
+class LeonardoServiceSpec extends TestKit(ActorSystem("leonardotest")) with FlatSpecLike with Matchers with BeforeAndAfterAll {
+  import system.dispatcher
 
-class LeonardoServiceSpec extends FlatSpec with Matchers {
-  import scala.concurrent.ExecutionContext.Implicits.global
-
-  implicit val system = ActorSystem("leonardotest")
+  override def afterAll(): Unit = {
```

Comment:
Akka `TestKit` wants you to clean up the `ActorSystem` yourself; `ScalatestRouteTest` does it for you. Go figure.